src/main/kotlin/com/openlattice/launchpad/configuration/IntegrationRunner.kt

```diff
@@ -195,11 +271,11 @@ class IntegrationRunner {
         private fun unsafeExecuteSql(
                 sql: String,
                 integrationName: String,
-                destination: LaunchpadDestination,
+                destination: DataLake,
                 integration: Integration,
                 start: OffsetDateTime
         ) {
-            destination.hikariDatasource.connection.use { connection ->
+            destination.getHikariDatasource().connection.use { connection ->
```

Comment:
🤔

Reply:
It's b/c hikariDatasource is created on demand and not stored as a field in the object in memory

Reply:
ooooohh i see